File: Samples-Xamarin/Fragments/Dataform/DataFormLabelPositionsFragment.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Util;
using Android.Views;
using Android.Widget;
using Com.Telerik.Widget.Dataform.Visualization;
using Com.Telerik.Widget.Dataform.Engine;

namespace Samples
{
	public class DataFormLabelPositionsFragment : Android.Support.V4.App.Fragment, ExampleFragment
	{
		RadDataForm dataForm;
		Person person;

		public String Title() {
			return "Label Positions";
		}

		public override View OnCreateView (LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
		{
			ViewGroup layoutRoot = (ViewGroup)inflater.Inflate(Resource.Layout.fragment_dataform_label_positions, container, false);

			dataForm = (RadDataForm)layoutRoot.FindViewById(Resource.Id.dataForm);
			person = new Person();
			dataForm.SetEntity(person);
			dataForm.LayoutManager = (new DataFormLinearLayoutManager(Activity));

			// >> data-form-label-positions
			dataForm.LabelPosition = LabelPosition.Left;
			// << data-form-label-positions

			return layoutRoot;
		}
	}
}

File: Samples-Java/Samples/src/main/java/fragments/dataform/DataFormLabelPositionsFragment.java
package fragments.dataform;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.telerik.android.sdk.R;
import com.telerik.widget.dataform.engine.LabelPosition;
import com.telerik.widget.dataform.visualization.DataFormLinearLayoutManager;
import com.telerik.widget.dataform.visualization.RadDataForm;

import activities.ExampleFragment;

public class DataFormLabelPositionsFragment extends Fragment implements ExampleFragment {
    RadDataForm dataForm;
    Person person;

    @Override
    public String title() {
        return "Label Positions";
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        ViewGroup layoutRoot = (ViewGroup)inflater.inflate(R.layout.fragment_dataform_label_positions, container, false);

        dataForm = (RadDataForm)layoutRoot.findViewById(R.id.dataForm);
        person = new Person();
        dataForm.setEntity(person);
        dataForm.setLayoutManager(new DataFormLinearLayoutManager(getContext()));

        // >> data-form-label-positions
        dataForm.setLabelPosition(LabelPosition.LEFT);
        // << data-form-label-positions

        return layoutRoot;
    }
}
